core/templates/core/about.html

```diff
@@ -12,31 +12,28 @@
         <div class="container">
             <div class="static">
                 <h1 class="header">About</h1>
-                <p aria-labels="about-list-1">GREAT.gov.uk is the UK government’s single destination for information on trade, investment, tourism and education provided by:</p>
+                <p>Great.gov.uk brings together international trade and investment information and services from across government. It is maintained by the <a href="https://www.gov.uk/government/organisations/department-for-international-trade" title="Department for International Trade - GOV.UK">Department for International Trade (DIT)</a>.</p>
+				<p>DIT is the government department responsible for promoting and financing international trade and investment, and championing free trade.</p>
```

Comment:
a bit nit-picky but can you keep the indentation consistent so the p tags here are lined up. just makes it easier to read

Reply:
yep - using soft wrap to view the text easier and got a bit mixed up. should be good now.